AppCenterCrashes/AppCenterCrashes/MSCrashes.h

```diff
@@ -89,6 +89,13 @@ typedef NS_ENUM(NSUInteger, MSUserConfirmation) {
  */
 + (BOOL)hasCrashedInLastSession;
 
+/**
+ * Check if the app received memory warning in the last session.
+ *
+ * @return Returns YES is the app received memory warning in the last session.
+ */
++ (BOOL)hadMemoryWarningInLastSession;
```

Comment:
Is the name confirmed by the PM?

Reply:
+1 - since the API we are referring to is called `didReceiveMemoryWarning`, maybe it would make sense to use `didReceiveMemoryWarningInLastSession` or `receivedMemoryWarningInLastSession`

Reply:
We have `hasCrashedInLastSession` in all platforms.
The most consistent would be `hasReceivedMemoryWarningInLastSession` according to me.